underlay/src/main/resources/config/datamapping/aouRT/entitygroup/procedurePerson/entityGroup.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "procedurePerson",
```

Comment:
Can we name `procedureSnomedPerson` since it represents the SNOMED hierarchy relationship with person